source/_includes/installation/operating_system.md

```diff
@@ -99,23 +126,20 @@ _Select and copy the URL or use the "copy" button that appear when you hover it.
 
 ### Start up your {{site.installation.types[page.installation_type].board}}
 
+{% if page.installation_type == 'generic-x86-64' %}
+1. If you used your Desktop system to write to your installation media, install the installation media ({{site.installation.types[page.installation_type].installation_media}}) to the target system. Otherwise shutdown the live operating system and make sure to remove the USB flash drive you have been using for the live system.
```

Comment:
```suggestion
1. If you used your Desktop system to write to your installation media, install the installation media ({{site.installation.types[page.installation_type].installation_media}}) to the target system. Otherwise, shutdown the live operating system and make sure to remove the USB flash drive you have been using for the live system.
```